app/lib/actions/pulp/repository/sync.rb

```diff
@@ -30,8 +31,6 @@ def invoke_external_task
             sync_options[:feed] = input[:source_url] if input[:source_url]
             sync_options[:validate] = !(SETTINGS[:katello][:pulp][:skip_checksum_validation])
 
-            sync_options.merge(input[:options]) if input[:options]
```

Comment:
@jlsherrill, merge! also appears to work in this case.  Updated.

Honestly, not sure how it passed before, perhaps something changed in the environment.